### ISSUE TITLE
 Generic.WhiteSpace.DisallowSpaceIndentSniff allows multiple whitespaces after tab in doc comments

### ISSUE THREAD

**Issue by Konafets** [squizlabs/PHP_CodeSniffer#393]
Given the following code, which is mainly taken from the UnitTest:

```php 
// Doc comments are indent with tabs and one space
// [tab]/**
// [tab][space]*
// [tab][space][space]*
// [tab][space]*/

    /**
     *
      *
     */
```
Is this behavior intended? If yes, can we introduce a variable to define the allowed whitespaces? ```$multipleWhitespacesAllowed = true|false;```

And what is the reason that this sniff ignores file/class level doc comments?

**Comment by gsherwood** [comment]
You probably want to look at the Squiz.Commenting.DocCommentAlignment sniff. Running that over your sample code produces:

```
$ phpcs temp.php --standard=Squiz --sniffs=Squiz.Commenting.DocCommentAlignment

FILE: temp.php
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 4 | ERROR | [x] Expected 1 space(s) before asterisk; 2 found
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------

Time: 8ms; Memory: 1.75Mb
```

**Comment by aik099** [comment]
It turned out, that I was using that sniff in my standard as well, but totally forgot about this. I should have recommended it myself as well.

**Comment by Konafets** [comment]
Thanks for recommending this sniff. We will use this sniff from now on too.
